src/ui/windows/TogglDesktop/TogglDesktop/ui/windows/MiniTimerWindow.xaml.cs

```diff
@@ -43,12 +42,9 @@ static void OnClosing(object sender, CancelEventArgs e)
 
         protected override void OnMouseRightButtonDown(MouseButtonEventArgs e)

```

Comment:
I don't see a point in this method anymore. It tweaks the placement slightly, but
- it can be tweaked in XAML anyway
- I don't see a benefit in doing so, the default way works fine

I think this is just a leftover from some old logic and it can be safely removed.

Reply:
The default way would be different from the current: it would place the menu on mouse point. Is this OK for us?

Reply:
I think it's totally fine. Do you see any downside with this?

Reply:
I think it would be even better. There is a strange bug in placement now:
![image](https://user-images.githubusercontent.com/11901598/102060381-64f3a080-3df2-11eb-81b3-f3899f7e0df2.png)
And if we go for mouse point placement, it will be fixed automatically.